assets/static/js/dataset-submission.js

```diff
@@ -325,11 +325,10 @@ $(function() {
                 if ([401].includes(jqXHR.status)) {
                     loggedOutPopup.show();
                 } else {
-                    if (jqXHR.status !== 0) {
-                        message = jqXHR.responseJSON == null ? errorThrown: jqXHR.responseJSON.message;
+                    if (jqXHR.status === 0) {
+                        errorPopup.show();
+                        $("#errorPopupText").html(message);
```

Comment:
The message can be hard-coded here, it will never we anything else.
This is going to get refactors in another branch anyway tho.